spec/controllers/saml_idp_controller_spec.rb

```diff
@@ -1489,10 +1489,31 @@ def name_id_version(format_urn)
             expect(response.status).to eq(200)
             expect(@analytics).to have_logged_event('SAML Auth', hash_including(analytics_hash))
           end
+
+          context 'Certificate does not match because of order bug' do
```

Comment:
I am confused because this context is under a "Matching certificate" context!

Reply:
ok, i can update the context

Reply:
(it's the scenario where the SP has a registered certificate that matches the certificate in the requets, but the idp is not finding it as a matching certificate because of the namespace bug)